src/OpenLoco/OpenLoco.h

```diff
@@ -33,6 +33,14 @@ namespace OpenLoco
 
     extern const char version[];
 
+    namespace GameSpeed
+    {
+        constexpr uint8_t Normal = 0;           // 0x00439A70 (speed: 0)
+        constexpr uint8_t FastForward = 1;      // 0x00439A93 (speed: 1)
+        constexpr uint8_t ExtraFastForward = 2; // 0x00439AB6 (speed: 2)
```

Comment:
I would really prefer the address comments to stay with the `setGameSpeed` function, as originally suggested. (This function is in OpenLoco.cpp as of this PR.)

Reply:
Oh right, sorry I didn't quite understand you wanted them in that specific spot, I'll move them back